omx/gstomxaacenc.c

```diff
@@ -53,6 +53,7 @@ enum
 #define DEFAULT_AAC_TOOLS (OMX_AUDIO_AACToolMS | OMX_AUDIO_AACToolIS | OMX_AUDIO_AACToolTNS | OMX_AUDIO_AACToolPNS | OMX_AUDIO_AACToolLTP)
 #define DEFAULT_AAC_ER_TOOLS (OMX_AUDIO_AACERNone)
 
+#define INPUT_BUFFER_NUM_SAMPLES 1024
```

Comment:
Consider using a more contextualized name for the macro. Instead of INPUT_BUFFER_NUM_SAMPLES something like OMXAACENC_INBUF_NSAMPLES ?

Reply:
I changed it to GST_OMX_AAC_ENC_INBUF_NSAMPLES and moved it to the helper defs for TI DM81xx in gstomx.h